app/_locales/en/messages.json

```diff
@@ -2425,7 +2425,7 @@
     "message": "Select a higher gas fee to accelerate the processing of your transaction.*"
   },
   "selectAccounts": {
-    "message": "Select account(s)"
+    "message": "Select the account(s) to use on this site"
```

Comment:
whats the process for getting the translations for other languages? or should the translation stay the same?

Reply:
I suppose the idea is to have this change and usher in changes to the other locales once we relay that message to the translators? @rekmarks thoughts?

Reply:
Yeah, this change is fine, and we will deal with updating the translations out of band.